app/models/host/base.rb

```diff
@@ -59,8 +59,6 @@ def self.taxonomy_conditions
                                     :domain, :domain_id, :domain_name,
                                     :lookup_values_attributes].freeze
 
-    after_initialize :suggest_default_pxe_loader
-
```

Comment:
I noticed this is not needed as it's defined in `PxeLoaderSuggestion` concern. Removing to match other three instances of this (managed_host and hostgroup files).